views/UserProfile.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+        <link rel="shortcut icon" href="img/Graphicloads-Transport-Bus-4.ico" type="image/x-icon">
+        <link rel="icon" href="img/Graphicloads-Transport-Bus-4.ico" type="image/x-icon">
+
+        <link href="https://cdn.metroui.org.ua/v4/css/metro-all.min.css" rel="stylesheet">
+        <link href="css/mainui.css" rel="stylesheet">
+    </head>
+    <body>
+        <div class="container">
+            <aside class="sidebar z-2 csidebar" data-role="sidebar" data-toggle="#sidebar-toggle" id="sb1">
+                <div class="sidebar-header color-effect csidebar-header">
+                    <a href="/" class="fg-white sub-action" onclick="Metro.sidebar.close('#sb1'); return false;"><span
+                            class="mif-arrow-left mif-2x"></span></a>
+                    <span class="title csidebar-title fg-white">Name</span>
+                    <span class="subtitle csidebar-subtitle fg-white">USN</span>
+                </div>
+                <ul class="sidebar-menu">
+                    <li><a><span class="mif-home fg-red icon"></span>Home</a></li>
+                    <li><a><span class="mif-books icon"></span>Guide</a></li>
+                    <li><a><span class="mif-files-empty icon"></span>Examples</a></li>
+                    <li class="divider"></li>
+                    <li><a><span class="mif-enter fg-green icon"></span>Login</a></li>
+                    <li><a><span class="mif-exit fg-blue icon"></span>Log Out</a></li>
+                </ul>
+            </aside>
+            <div class="aaa p-ab">
+                <div class="app-bar cappbar pos-fixed color-effect z-1" data-role="appbar" id="app-bar-1">
+                    <button class="app-bar-item cappbar-item c-pointer" id="sidebar-toggle"><span class="mif-menu cappbar-menu fg-white"></span></button>
+                    <h1 class="brand cappbar-header no-hover fg-white mx-auto">Transport Management System</h1>
+                </div>
+
+                <div class="p-4">
+                    <!--content goes here-->
+                </div>
+            </div>
+
+
+
+        <div class="row">
+        	<section class="basic-info">
+            	<div class="row">
+                	<div class="col-xs-12">
+                    	<div class="profilePic">
+                    		<div class="pos-relative pos-top-center">
+                        		<img src="IMG/dummy_5.png" alt="Profile Picture" class="img-container rounded center">
+                    		</div>
+                    	</div>
+                	</div>
+            	</div>
+        	</section>
+    	</div>
+
+        <br><br>
+        <div class="row">
+        	<h2>Allocated Trips</h2>
+        </div>	
+
+        <br><br>
+
+        <div class="row">
```

Comment:
Can you do like [this](https://www.w3schools.com/howto/howto_css_menu_horizontal_scroll.asp) and refer to the image in trello